elements/bulbs-video/components/revealed.js

```diff
@@ -55,6 +55,9 @@ export default class Revealed extends React.Component {
     let specialCoverage = this.props.targetSpecialCoverage || 'None';
     let autoplayInViewBool = typeof this.props.autoplayInView === 'string';
 
+    // Allowing creativeSize to be passed in in root.js
+    let creativeSize = this.props.creativeSize;
```

Comment:
I would make this an optional attribute, with a default value of `640x480`, so simply change this to: 

```
let creativeSize = this.props.creativeSize || '640x480';
```